src/shogun/lib/GPUMatrix.h

```diff
@@ -41,6 +41,7 @@
 
 #include <shogun/lib/common.h>
 #include <memory>
+#include <viennacl/linalg/matrix_operations.hpp>
```

Comment:
Nope, no element access for GPU structures (forces user to transfer back to cpu first)